src/AuthContext.tsx

```diff
@@ -146,8 +145,6 @@ export const AuthProvider = (props: AuthProviderProps) => {
         dispatch({ authInfo })
     }, [dispatch])
 
-    const activeOrgFn = props.getActiveOrgFn || loadOrgSelectionFromLocalStorage
```

Comment:
Since we are just deprecating this, we can't remove it. It needs to continue to function but we'll log the warning for now.

Since the AuthContext type is private, you can also rename this to deprecatedActiveOrgFn

Reply:
Sure, I'm just not sure it should have any functionality now, though I understand the need for backward compatibility. At one point, we would want users to be unable to use this. 

I will also add a TODO to eventually completely remove this.